prometheus-metrics-ghc/src/Prometheus/Metric/GHC.hs

```diff
@@ -124,14 +128,61 @@ ghcCollectors = [
             CounterType
             parMaxBytesCopied
     ]
+#else
+ghcCollectors :: [IO [SampleGroup]]
+ghcCollectors = [
+      statsCollector "gcs" "Total number of GCs" CounterType gcs
+    , statsCollector "major_gcs" "Total number of major (oldest generation) GCs" CounterType major_gcs
+    , statsCollector "allocated_bytes" "Total bytes allocated" CounterType allocated_bytes
```

Comment:
Likewise, this should be `allocated_bytes_total`, since this is (presumably) an accumulating count with unit.